openapi/openapi.yaml

```diff
@@ -1287,16 +1292,31 @@ components:
           type: string
       required: [title, url]
 
-
-    User:
+    UserMetadataInDB:
       type: object
       properties:
-        email:
+        id:
+          description: |
+            identical to the sub in Cognito, which is a unique identifier for
+            the authenticated user.
           type: string
         preferred_username:
+          description: identical to the preferred_username in Cognito.
           type: string
-        username:
-          type: string
-        api_key:
-          type: string
-      required: [email, preferred_username, username, api_key]
+      required: [id, preferred_username]
+
+    UserMetadata:
```

Comment:
I think we can just call it `User`.  We only have one type of user model so it is pretty clear this is just all the attributes of the user. WDYT?

Reply:
Sounds good!